R/app-server.R

```diff
@@ -281,27 +281,46 @@ app_server <- function(input, output, session) {
     })
 
     ## Successes box
-    output$successes <- renderUI({
+    observe({
       successes <- purrr::map_lgl(res(), function(x) {
         inherits(x, "check_pass")
       })
-      report_results(res()[successes], emoji_prefix = "check")
+      output$successes <- renderUI({
+        report_results(res()[successes], emoji_prefix = "check")
+      })
+      output$num_success <- renderText({
+        paste0("Successes (", as.character(sum(successes)), ")")
+      })
     })
 
     ## Warnings box
-    output$warnings <- renderUI({
+    observe({
       warnings <- purrr::map_lgl(res(), function(x) {
         inherits(x, "check_warn")
       })
-      report_results(res()[warnings], emoji_prefix = "warning", verbose = TRUE)
+      output$warnings <- renderUI({
+        report_results(res()[warnings],
```

Comment:
Tidyverse style would put the `res()[warnings],` here on a new line

Reply:
Thanks! I'll fix that and also install the dev version of styler. Perhaps it will help me catch these better.